docs/content/getting_started/migration_guide.md

```diff
@@ -630,3 +669,26 @@ In AWS PDK, these classes have been moved out from `Handlers` as their own stand
         }
     }
     ```
+
+## Troubleshooting
+### Missing Polyfill
+
+If you see this error:
+
+```
```

Comment:
Probably best not to make Greg famous 😉

Reply:
Also, this can be caused by using the ts runtime in a website- might be worth mentioning the hooks lib should be used in a website even if you use the raw client, since the runtime has all the server side handlers so is less suitable for web.

Reply:
Good catch - resolved and added section regarding website -> runtime.